docs-2.0/3.ngql-guide/10.tag-statements/3.alter-tag.md

```diff
@@ -8,6 +8,12 @@
 
 - Before you alter properties for a tag, make sure that the properties are not indexed. If the properties contain any indexes, the conflict error `[ERROR (-1005)]: Conflict!` will occur when you `ALTER TAG`. For more information on dropping an index, see [DROP INDEX](../14.native-index-statements/6.drop-native-index.md).
 
+- Make sure that the added attribute name is not the same as an existing or deleted attribute name, otherwise the added attribute will fail.
```

Comment:
```suggestion
- Make sure that the added property name is not the same as an existing or deleted property name, otherwise the added property will fail.
```